### ISSUE TITLE
[FEAT] 결측치 대체 및 피쳐엔지니어링

### ISSUE THREAD

**Issue by HoChangLee98** [HoChangLee98/my_project#7]
## Contents

* 평균값 이외에 결측치를 대체할 다른 방법 찾아보기
* 피쳐엔지니어링 고려하기

## To Do

- [x] 결측치 대체
- [x] 피쳐엔지니어링

## Result
**[version 4]**
1. xgboost 
   * `nan` : area_mean
   * `classification_method` : False
   * `log transform` : False 
   * valid set : 46.720971003986335
   * test set : 46.3929167595
-----
2. xgboost 
    * `nan` : area_mean
    * `classification_method` : True
    * `log transform` : False
    * valid set : 
    * test set : 
-----
3. xgboost 
    * `nan` : area_mean
    * `classification_method` : False
    * `log transform` : True
    * valid set : 47.39990355223736
    * test set : 46.9066107896
-----
4. lightgbm
    * `nan` : area_mean
    * `classification_method` : False
    * `log transform` : False
    * valid set : 52.445968562416745
    * test set : 
-----
5. lightgbm
    * `nan` : area_mean
    * `classification_method` : True
    * `log transform` : False
    * valid set : 
    * test set : 
-----
6. lightgbm
    * `nan` : area_mean
    * `classification_method` : True
    * `log transform` : True
    * valid set : 
    * test set : 
-----
7. randomforest
    * `nan` : area_mean
    * `classification_method` : False
    * `log transform` : False
    * valid set : 
    * test set : 
-----
8. randomforest
    * `nan` : area_mean
    * `classification_method` : True
    * `log transform` : False
    * valid set : 
    * test set : 
-----
9. randomforest
    * `nan` : area_mean
    * `classification_method` : True
    * `log transform` : True
    * valid set : 
    * test set :

**Comment by HoChangLee98** [comment]
**[version 4]**
1. xgboost (binary : lightgbm , nan : area , log : T)
   * valid set : 50.34991165229915
   * test set : 49.4300257635
-----
2. xgboost 
    * `nan` : area_mean
    * `classification_method` : False
    * `log transform` : False
    * valid set : 
    * test set :

**Comment by HoChangLee98** [comment]
* 10/9 
   * 결측치 대체 : 항구명 별 평균
   * 피쳐엔지니어링 : `wind_speed`
